telegram/inputfile.py

```diff
@@ -66,6 +67,9 @@ def __init__(self, data):
         elif 'certificate' in data:
             self.input_name = 'certificate'
             self.input_file = data.pop('certificate')
+        elif 'video_note' in data:
+            self.input_name = 'video_note'
+            self.input_file = data.pop('video_note')
```

Comment:
I wonder - we have a lot of branching here. Should we move to a dict here? It's much more pythonic.